### PR TITLE
Consent flow newsletter counter

### PR DESCRIPTION
**Pull request by walaura**
## What does this change?
Adds a clicked checkbox counter to the email preferences step of the consent flow

## What is the value of this and can you measure success?
This is part of a series of measures to encourage our users to tick more checkboxes

screen shot 2017-11-30 at 16 29 06

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/identity/consent-wizard.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+import fastdom from 'lib/fastdom-promise';
+
+import loadEnhancers from './modules/loadEnhancers';
+import { newsletterCheckboxClassName } from './consents';
+import { wizardPageChangedEv } from './wizard';
+
+const getClickedCheckboxes = (
+    checkboxesEl: Array<HTMLLabelElement>
+): Promise<void> =>
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/identity/consent-wizard.js]: It should probably be `Promise<HTMLLabelElement[]>`

File: static/src/javascripts/projects/common/modules/identity/consent-wizard.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    fastdom.read(
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/identity/consent-wizard.js]: No reason to use fastdom here.

File: static/src/javascripts/projects/common/modules/identity/consent-wizard.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        () =>
+            checkboxesEl.filter(
+                (checkboxEl: HTMLLabelElement) =>
+                    checkboxEl.control instanceof HTMLInputElement &&
+                    checkboxEl.control.checked
+            ).length
+    );
+
+const getEmailCheckboxes = (): Promise<Array<HTMLLabelElement>> =>
+    fastdom.read(() => [
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/identity/consent-wizard.js]: ditto.

Comment (regiskuckaertz) [reply static/src/javascripts/projects/common/modules/identity/consent-wizard.js]: Nope, only when reading properties that are layout-related. See the list [here](https://gist.github.com/paulirish/5d52fb081b3570c81e3a).

**Review by mariogalic — APPROVED** [review]

**Pull request merged**